docs/user_guides/fs/transformation_functions.md

```diff
@@ -30,8 +36,11 @@ The `@udf` decorator in Hopsworks creates a metadata class called [`HopsworksUdf
 | datetime.date            |
 | datetime.time            |
 
+The `drop` argument is optional and specifies the input arguments to remove from the final output after all transformation functions are applied. By default, all input arguments are retained in the final transformed output. 
 
-Hopsworks supports four types of transformation functions:
+The `mode` argument controls the execution mode of transformation functions and accepts three values: `default`, `python`, or `pandas`. The `default` mode assumes the function can be executed as both a Python and Pandas UDF, the transformation function in this mode is  executed as a Python UDF for online inference and as a Pandas UDF for batch inference and training dataset generation. Setting mode to `pandas` forces the function to always run as a Pandas UDF, while setting the mode to `python` ensures it always runs as a Python UDF.
```

Comment:
as both a -> as either a Python or Pandas UDF

Reply:
, the transformation function -> . The transformation function
 is  executed -> too many spaces
 generation -> creation
 
 Note: check if our docs use the term dataset generation or creation. I would use creation, but be consistent with what is in our docs.

Reply:
Updated to creation everywhere in the text.